src/ConfigFactory.Avalonia/Builders/DropdownControlBuilder.cs

```diff
@@ -0,0 +1,43 @@
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Layout;
+using ConfigFactory.Core;
+using ConfigFactory.Core.Attributes;
+using System.Reflection;
+
+namespace ConfigFactory.Avalonia.Helpers;
+
+internal static class DropdownBuilder
+{
+    internal static ComboBox Build(IConfigModule context, PropertyInfo propertyInfo, DropdownConfigAttribute dca)
+    {
+        var combobox = new ComboBox
+        {
```

Comment:
Variables should be statically typed and curly braces are inline (to match the rest of the codebase)

```suggestion
        ComboBox combobox = new() {
```